src/Shared/AppModels/AppDebug.cs

```diff
@@ -0,0 +1,167 @@
+
+        private void SetWindowSize()
+        {
+            WindowSizeStr = $"Current size is: Height: {Window.Current.Bounds.Height}, Width: {Window.Current.Bounds.Width}";
+        }
+
+        private async void Logger_MessageReceived(object sender, LoggerMessageEventArgs e)
+        {
+            await _semaphoreSlim.WaitAsync();
+
+            var formatedMessage = LogFormatter.GetFormattedLogMessage(e);
+
+            if (AppLogs == null)
+                return;
+
+            await CoreApplication.MainView.CoreWindow.Dispatcher.RunAsync(CoreDispatcherPriority.Normal,
+            () =>
+            {
+                AppLogs.Add(formatedMessage);
+            });
+
+            _semaphoreSlim.Release();
+        }
+
+        [RelayCommand]
+        private async void OpenLogFolder()
+        {
+            var logsLocation = LogFormatter.LogFolderPath;
+            var folder = await StorageFolder.GetFolderFromPathAsync(logsLocation);
+
+            await Launcher.LaunchFolderAsync(folder);
+        }
+
+        [RelayCommand]
+        private async void DeleteUserData()
+        {
+            var contentDialog = new ContentDialog()
+            {
+                Title = "Delete user data",
+                Content = "Are you sure you want to delete user data? If you hit yes, the app will restart.",
+                PrimaryButtonText = "Yes",
+                SecondaryButtonText = "No",
+                PrimaryButtonCommand = PerformUserDataDeletionCommand
+            };
+
+            await contentDialog.ShowAsync();
+        }
+
+        private async void PerformUserDataDeletion()
+        {
+            try
+            {
+                var localDataFolder = ApplicationData.Current.LocalFolder.Path;
+                var localCache = ApplicationData.Current.LocalCacheFolder.Path;
+
+                var folder = await StorageFolder.GetFolderFromPathAsync(localDataFolder);
+                await folder.DeleteAsync();
+
+                folder = await StorageFolder.GetFolderFromPathAsync(localCache);
+                await folder.DeleteAsync();
+
+                var result = await CoreApplication.RequestRestartAsync("Application Restart Programmatically.");
+
+                if (result == AppRestartFailureReason.NotInForeground ||
+                   result == AppRestartFailureReason.RestartPending ||
+                   result == AppRestartFailureReason.Other)
+                {
+                    var msgBox = new MessageDialog("Restart Failed. Please restart the app manually.", result.ToString());
+                    await msgBox.ShowAsync();
+                }
+            }
+            catch (Exception)
+            {
+                var msgBox = new MessageDialog("Couldn't delete the user data.");
+                await msgBox.ShowAsync();
+            }
+        }
+
+        private string SizeSuffix(long value, int decimalPlaces = 1)
```

Comment:
Code style and formatting for this method is incorrect for this project. Let's do some basic cleanup.